demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -302,6 +303,10 @@ def _is_pack_meta_file_structure_valid(self):
                 if self._add_error(Errors.pack_metadata_field_invalid(), self.pack_meta_file):
                     return False
 
+            if len(description_name) > MAXIMUM_DESCRIPTION_FIELD_LENGTH:
+                if self._add_error(Errors.pack_metadata_long_description(), self.pack_meta_file):
+                    return False
```

Comment:
Is there a reason for this limitation? might we want to reduce this to a warning?